spec/support/templates/post_form.rb

```diff
@@ -0,0 +1,5 @@
+require 'reform/form'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.